From 725672e8903791f70661699c03921d560b0a1c16 Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Mon, 8 Jun 2026 14:03:08 +0200
Subject: [PATCH] fixes

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index e6f8c18..f3c12f6 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -25,7 +25,7 @@ classifiers = [
     "Programming Language :: Python :: 3.14",
 ]
 dependencies = [
-    "numpy>=1.23.5,<=2.4.3",
+    "numpy>=1.23.5,<=2.4.6",
     "pint>=0.18,<=0.25.3",
 ]
 dynamic = ["version"]
@@ -37,7 +37,7 @@ Repository = "https://github.com/pyiron/pyiron_dataclasses"
 
 [project.optional-dependencies]
 convert = [
-    "h5io_browser>=0.1.2,<=0.2.12",
+    "h5io_browser>=0.1.2,<=0.2.15",
 ]
 
 [tool.ruff]
